### ISSUE TITLE
[IE11] Files Mediaviewer does not open images

### ISSUE THREAD

**Issue by davitol** [owncloud/files_mediaviewer#90]
### Steps to reproduce

1 - Upload a photo eg. ```jpeg``` or ```png``` with Internet Explorer 11 as browser
2 - Try to open the file

### Expected behaviour

The photo is opened

### Actual behaviour

No action is triggered when clicking on the photo. No logs are spotted

### Server configuration

**Operating system**:
Ubuntu 16.04

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
7.1

**ownCloud version:**

ownCloud 10.2.1 
**Updated from an older ownCloud or fresh install:**
Fresh

**Are you using external storage, if yes which one:** 
No

**Are you using encryption:** 
No

**Logs**

```
No logs in owncloud.log
```

### Client configuration

**browser**
Internet Explorer 11.953.14393 in Windows 10

@felixheidecke JFYI

**Comment by davitol** [comment]
```
Enabled:
  - activity: 2.5.0
  - admin_audit: 1.0.3
  - comments: 0.3.0
  - configreport: 0.2.0
  - dav: 0.4.0
  - enterprise_key: 0.2.0
  - federatedfilesharing: 0.4.0
  - federation: 0.1.0
  - files: 1.5.2
  - files_external: 0.7.1
  - files_mediaviewer: 1.0.0
  - files_pdfviewer: 0.11.0
  - files_sharing: 0.11.0
  - files_texteditor: 2.3.0
  - files_trashbin: 0.9.1
  - files_versions: 1.3.0
  - files_videoplayer: 0.10.1
  - firewall: 2.8.0
  - firstrunwizard: 1.2.0
  - market: 0.5.0
  - notifications: 0.5.0
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - systemtags_management: 0.3.1
  - templateeditor: 0.4.0
  - theme-enterprise: 2.1.2
  - updatenotification: 0.2.1
  - windows_network_drive: 0.7.4
  - workflow: 0.3.0
Disabled:
  - announcementcenter
  - customgroups
  - encryption
  - external
  - files_antivirus
  - files_classifier
  - files_external_dropbox
  - files_external_ftp
  - files_ldap_home
  - gallery
  - guests
  - impersonate
  - oauth2
  - password_policy
  - ransomware_protection
  - sharepoint
  - twofactor_totp
  - user_external
  - user_ldap
  - user_shibboleth
  - wopi
```

**Comment by felixheidecke** [comment]
A fix has been pushed to `release-1.0.0`

**Comment by davitol** [comment]
Fixed by https://github.com/owncloud/files_mediaviewer/commit/d733a11fe6d893ad532ab083bf18197ff3dc453b
